tools/ld-chroma-decoder/decoder.h

```diff
@@ -83,14 +83,16 @@ class Decoder {
         LdDecodeMetaData::VideoParameters videoParameters;
         qint32 topPadLines;
         qint32 bottomPadLines;
-    };
+        QString pix_fmt = "RGB48";
```

Comment:
pixelFormat, not pix_fmt.

It would be better to make this an enumerated type, like the other choose-one-of-several options in the Configuration structs, not a string - QString does all sorts of horrid implicit-sharing stuff internally, and comparing QStrings later on isn't going to be very efficient (plus the compiler can't warn you if you forget to handle one of the possibilities).

Reply:
I copied pix_fmt libavformat and didn't think twice about a proper C++ name.